.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -392,36 +378,80 @@ kernel_matrix_testing_run_tests_x64:
     - .kernel_matrix_testing_run_tests
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/system-probe_x64$DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_SYSPROBE_BUILDIMAGES
   tags: ["arch:amd64"]
-  needs: ["kernel_matrix_testing_setup_env_x64", "upload_dependencies_x64", "upload_system_probe_tests_x64", "upload_minimized_btfs_x64"]
+  needs:
+    [
+      "kernel_matrix_testing_setup_env_x64",
+      "upload_dependencies_x64",
+      "upload_system_probe_tests_x64",
+      "upload_minimized_btfs_x64",
+    ]
   timeout: 3h
   variables:
     ARCH: "x86_64"
   parallel:
     matrix:
-      - TAG: ["ubuntu_16.04", "ubuntu_18.04", "ubuntu_20.04", "ubuntu_22.04", "ubuntu_23.10", "amzn_4.14", "amzn_5.4", "amzn_5.10", "fedora_37", "fedora_38", "debian_10", "debian_11", "debian_12", "centos_79", "centos_8"]
+      - TAG:
+          [
+            "ubuntu_16.04",
+            "ubuntu_18.04",
+            "ubuntu_20.04",
+            "ubuntu_22.04",
+            "ubuntu_23.10",
+            "amzn_4.14",
+            "amzn_5.4",
+            "amzn_5.10",
+            "fedora_37",
+            "fedora_38",
+            "debian_10",
+            "debian_11",
+            "debian_12",
+            "centos_79",
+            "centos_8",
+          ]
         TEST_SET: ["no_tracersuite", "only_tracersuite"]
 
 kernel_matrix_testing_run_tests_arm64:
   extends:
     - .kernel_matrix_testing_run_tests
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/system-probe_arm64$DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_SYSPROBE_BUILDIMAGES
   tags: ["arch:arm64"]
-  needs: ["kernel_matrix_testing_setup_env_arm64", "upload_dependencies_arm64", "upload_system_probe_tests_arm64", "upload_minimized_btfs_arm64"]
+  needs:
+    [
```

Comment:
for multi-line arrays, I prefer the `-` format, but we haven't really standardized our Gitlab YAML on either format.